yarn-project/noir-libs/noir-aztec/src/abi.nr

```diff
@@ -296,7 +296,7 @@ struct PublicCircuitPublicInputs {
     new_l2_to_l1_msgs: [Field; crate::abi::MAX_NEW_L2_TO_L1_MSGS_PER_CALL],
     unencrypted_logs_hash: [Field; NUM_FIELDS_PER_SHA256],
     unencrypted_log_preimages_length: Field,
-    block_hash_data: BlockHashData, // TODO: This is not present in cpp or ts, do we need to include it?
+    block_data: ConstantBlockData, // TODO: This is not present in cpp or ts, do we need to include it?
```

Comment:
Is this still a todo? If so can you add issue for it.

Reply:
No longer a todo, however i will change the name to match exactly